rvps/build.rs

```diff
@@ -1,4 +1,4 @@
-use std::process::exit;
+use std::process::{exit, Command};
```

Comment:
`make lint` complains about the opposite: `error: unused import: `Command``

Reply:
Yeah - it's related with checking in-toto feature with is by default not set -> changed approach, should be ok now